packages/components/transfer/src/token.ts

```diff
@@ -22,7 +22,7 @@ interface TransferContext {
   sourceCheckableListRef: Ref<ɵCheckableListInstance | undefined>
   targetCheckableListRef: Ref<ɵCheckableListInstance | undefined>
   showSelectAll: ComputedRef<boolean>
-  getRowKey: GetRowKey
+  getRowKey: ComputedRef<GetKeyFn>
```

Comment:
```suggestion
  getKey: ComputedRef<GetKeyFn>
```

对外暴漏的 API 名称统一为这个吧，内部不想改可以不改。